target_chains/starknet/sdk/js/.eslintrc.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  root: true,
+  parser: "@typescript-eslint/parser",
+  plugins: ["@typescript-eslint"],
+  extends: ["eslint:recommended", "plugin:@typescript-eslint/recommended"],
+  rules: {
+    "@typescript-eslint/no-explicit-any": "off",
```

Comment:
do we need this anywhere? would be nice if we remove it

Reply:
Removed.